CHARTER.md

```diff
@@ -0,0 +1,176 @@
+
+- A voting representative must have attended two of the preceding three meetings to be counted as in good standing for the purposes of Quorum. Voting rights will be reinstated at the meeting after the voting representative has attended two of the prior three meetings. 
+        
+- Ideally decisions will be made based on consensus. If, however, any decision requires a vote to move forward, the voting representatives will vote on a one vote per voting representative basis.
+
+- Except as provided in Section 15.a. or elsewhere in this Charter, decisions by vote at a meeting will require a simple majority vote, provided quorum is met. Except as provided in Section 15.a. or elsewhere in this Charter, decisions by electronic vote without a meeting will require a majority of all voting representatives.
+  
+- In the event of a tied vote with respect to an action that cannot be resolved by the TAC, the TAC Representative may refer the matter to the Governing Board. In the event of a tied vote with respect to an action that cannot be resolved by the Governing Board, the chairperson may refer the matter to the LF for assistance in facilitating a decision.
+
+8. **Subsidiaries and Related Companies**
+    - Definitions:
+        - “Subsidiaries” means any entity in which a Member owns, directly or indirectly, more than fifty percent of the voting securities or voting membership interests of the entity in question;
+        
+        - “Related Company” means any entity which controls or is controlled by a Member or which, together with a Member, is under the common control of a third party, in each case where such control results from ownership, either directly or indirectly, of more than fifty percent of the voting securities or voting membership interests of the entity in question; and
+        
+        - “Related Companies” are entities that are each a Related Company of a Member.
+- Only the legal entity which has executed a Participation Agreement and its Subsidiaries will be entitled to enjoy the rights and privileges of such OpenSSF Membership. 
+
+- If a OpenSSF Member is itself a foundation, association, consortium, open source project, membership organization, user group or other entity that has members or sponsors, then the rights and privileges granted to such OpenSSF Member will extend only to the employee-representatives of such OpenSSF Member, and not to its members or sponsors, unless otherwise approved by the Governing Board in a specific case.
+
+- OpenSSF Membership is non-transferable, non-salable and non-assignable, except an OpenSSF Member may transfer its current OpenSSF Membership benefits and obligations to a successor of substantially all of its business or assets, whether by merger, sale or otherwise; provided that the transferee agrees to be bound by this Charter and the Bylaws and policies required by LF membership.
```

Comment:
Is this a good place to also added in Related Groups? Something to capture other groups already doing work in this space that the foundation can leverage and collaborate on, such as CNCF Security TAG and Kubernetes SIG-Security.  

There is some overlap of work between OpenSSF and these groups, particularly as we're driving towards the same goals (secure open source projects).  I'd like to us promote the need for collaboration.

Reply:
Charter changes require GB approval and, as far as I can tell from a quick read, this PR is just book-keeping. 

So I'd like this merged for transparency purposes first, and then we can discuss any changes/amendments. 

For what it's worth, I don't think the GB Charter is the right place to link to any external groups -- that could be a great addition to either TAC notes or specific WG pages where a WG collaborates with specific external groups.